xof/xof_test.go

```diff
@@ -214,6 +215,43 @@ func testReseed(t *testing.T, s kyber.XOFFactory) {
 	}
 }
 
+func TestResetNoSeed(t *testing.T) {
+	for _, impl := range impls {
```

Comment:
I think this is only testing `blake2xb`, no? Could you add `blake2xs` to the list of `impls` too?